docs/api.rst

```diff
@@ -2981,6 +2981,28 @@ of :class:`enum.Enum`.
 
         The guild scheduled event is only for a specific guild.
 
+.. class:: ThreadArchiveDuration
+
+    Represents the automatic archive duration of a thread in minutes.
+
+    .. versionadded:: 2.3
+
+    .. attribute:: hour
+
+        The thread will archive after an hour.
```

Comment:
sorry, missed this first time around. All of these should be after an xx of inactivity.